wpilibj/src/main/java/edu/wpi/first/wpilibj/XboxController.java

```diff
@@ -34,7 +34,7 @@
     kStart(8);
 
     @SuppressWarnings({"MemberName", "PMD.SingularField"})
-    private final int value;
+    public final int value;
```

Comment:
I think we do want to expose this, since it's an important part of the API and not an implementation detail; I've made a separate PR to do this, but will leave the change here until that one's merged.